src/tests/Button.test.js

```diff
@@ -11,4 +11,21 @@ describe('<Button />', () => {
     const div = document.createElement('div');
     ReactDOM.render(<Button />, div);
   });
+
+  it('should render a button', () => {
+    const button = shallow(<Button />);
+    // const button = header.find('.toggle');
+
+    expect(button.length).toEqual(1);
+  });
+
+  //trying to figure check a prop of the button that's orange
+  it.skip('should render a button that is orange', () => {
+    const wrapper = shallow(<Button />);
```

Comment:
Consider that when you pass a prop to a component you have to do something like this:
`<Component propName={propValue} />`
With that pattern in mind I would have approached passing a prop to the button like so:
`<Button orange={true} />`
Don't over think it, this is still react, we are just testing the results.
Also consider that the prop named orange is a Boolean prop, and should be expected to be true or false.